dango/types/src/account_factory/msg.rs

```diff
@@ -113,6 +113,12 @@ pub enum QueryMsg {
     /// Query a single user by username.
     #[returns(User)]
     User { username: Username },
+    #[returns(BTreeSet<Username>)]
+    UsersByKey {
```

Comment:
I'm going to rename this query. We have a query called `QueryMsg::User` that returns a `User` struct. If you call this query `UsersByKey` then I would expect it also returns `User` structs.

To avoid confusion, we can either rename this to `UsernamesByKey` or just `ForgotUsername`.